src/use_case/new_game/NewGameInteractor.java

```diff
@@ -1,4 +1,39 @@
 package use_case.new_game;
+import java.util.ArrayList;
+import entity.Game;
+import java.util.*;
+import entity.Werewolf;
+import entity.Villager;
+import use_case.data_access_interface.GameDataAccessInterface;
 
-public class NewGameInteractor implements NewGameInputBoundary {
+
+public class NewGameInteractor implements NewGameInputBoundary{
+
+    private Game game;
+
+    private GameDataAccessInterface gameData;
+
+    public NewGameInteractor(GameDataAccessInterface gameData){
+        this.game = new Game();
```

Comment:
You should get the game from gameData, see my interactor for the vote out use case. You also need a presenter as an attribute.

Reply:
Basically at the end of the execute prepare some sort of view for the presenter.

Reply:
Also make sure you save the game using the data access object, create output data and send it to the presenter. For example the end of my execute:
![image](https://github.com/yhz3/werewolf/assets/125763827/ade28460-ee3d-4680-93f4-bb1bbe55ccac)